src/core/CCArrayLabels.mli

```diff
@@ -129,23 +132,23 @@ val lookup : cmp:'a ord -> key:'a -> 'a t -> int option
       [Some i] ([i] the index of the key) otherwise. *)
 
 val lookup_exn : cmp:'a ord -> key:'a -> 'a t -> int
-(** Same as {!lookup}, but
+(** Like {!lookup}, but
     @raise Not_found if the key is not present. *)
 
 val bsearch : cmp:('a -> 'a -> int) -> key:'a -> 'a t ->
   [ `All_lower | `All_bigger | `Just_after of int | `Empty | `At of int ]
 (** [bsearch ?cmp key arr] finds the index of the object [key] in the array [arr],
     provided [arr] is {b sorted} using [cmp]. If the array is not sorted,
-    the result is not specified (may raise Invalid_argument).
+    the result is not specified (may @raise Invalid_argument).
```

Comment:
this is probably invalid ocamldoc. Have you tried compiling doc?

Reply:
Effectively, this is incorrect with ocamldoc. (no more printing after may).

@ deleted.